steps/aws-ec2-step-instances-describe/step.py

```diff
@@ -31,9 +31,26 @@ def instance_to_dict(ec2, instance):
   region_name=relay.get(D.aws.region),
   aws_session_token=session_token
 )
+
+list_of_filters = []
+try:
+  raw_filters = relay.get(D.filters)
+  for key, value in raw_filters.items():
+    f = {}
+    f['Name'] = key
+    f['Values'] = [value]
+    list_of_filters.append(f)
+except:
```

Comment:
If this exception is just for the "optional" filters spec property, perhaps something like this?
```python
except requests.exceptions.HTTPError as e:
    if e.response.status_code == 422:
        pass
    else:
        raise
```